mods/ts/weapons/superweapons.yaml

```diff
@@ -90,7 +90,7 @@ EMPulseCannon:
 		Range: 3c0
 		Duration: 250
 		Upgrades: empdisable
-		ValidTargets: Vehicle
+		ValidStances: Ally, Neutral, Enemy
```

Comment:
You don't need to explicitly set this, as `ValidStances` has this values as defaults.